dags/pytorch_xla/configs/pytorchxla_torchbench_config.py

```diff
@@ -272,16 +272,20 @@ def get_torchbench_gpu_config(
     image_family: resource.ImageFamily,
     accelerator_type: resource.GpuVersion,
     count: int,
-    gpu_zone: resource.Zone,
+    gpu_zone: Union[resource.Zone, List[resource.Zone]],
```

Comment:
Do you think it may be better if we just define it as `List[resource.Zone]]` even for a single zone? i.e. [`us-central2-b`]? so we don't need a if/else statement.

Reply:
Updated to list format.